assets/js/modules/ads/pax/services.js

```diff
@@ -49,11 +49,13 @@ const restFetchWpPages = async () => {
  * @since 1.126.0
  *
  * @param {Object} registry Registry object to dispatch to.
+ * @param {Object} _global  The global window object.
  * @return {Object} An object containing various service interfaces.
  */
-export function createPaxServices( registry ) {
+export function createPaxServices( registry, _global = global ) {
+	const { select, __experimentalResolveSelect: resolveSelect } = registry;
```

Comment:
This helps to avoid line breaking in repeated `registry.` calls below.

Also the alias for `__experimentalResolveSelect` is something we should start doing where it's used to minimize the number of replacements we need to make later since this is no longer experimental in recent versions. It's also substantially shorter :)